1080i/Includes_Info.xml

```diff
@@ -1327,7 +1327,7 @@
         <param name="is_web">false</param>
         <param name="visible">true</param>
         <definition>
-            <include content="Info_Line_VideoQuality_Resolution" condition="!Skin.HasSetting(InfoLine.DisableAdditionalTags) | !Skin.HasSetting(Info.EnablePillboxStyle)">
+            <include content="Info_Line_VideoQuality_Resolution" condition="[!Skin.HasSetting(InfoLine.DisableAdditionalTags) + Skin.HasSetting(Infoline.Source)] | !Skin.HasSetting(Info.EnablePillboxStyle)">
```

Comment:
If these are toggled individually, should get rid of the DisableAdditionalTags setting completely from the skin. It is now unnecessary and just complicates the code having to toggle it.

I get that it is nice to keep people's existing settings intact but the minor inconvenience is better now than have needlessly complicated code that has to be kept in longer term.